### PR TITLE
Nav button

### PR DESCRIPTION
**Pull request by yarosz**

### CODE REVIEW

File: client/components/Story.jsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import styled from 'styled-components';
+import { StyledDesc, NavHeader, SectionHeader, Caption, Img } from '../StyledDiv';
+
+const Story = (props) => {
+  const {story} = props;
```
Comment (quinnfinite) [inline client/components/Story.jsx:6]: not sure I understand these two lines. I'd be interested to chat about them later.

File: client/App.jsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import axios from 'axios';
+import styled from 'styled-components';
+import { createGlobalStyle as global } from 'styled-components';
+
+import Story from './components/Story.jsx';
+import Risks from './components/Risks.jsx';
+import Nav from './components/Nav.jsx';
+
+const GlobalStyle = global`
```
[the remark this answers is not in the document]
Comment (yarosz) [reply client/App.jsx:10]: That's a great suggestion - I just gave it a shot and it worked!

File: client/App.jsx [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+  }
+`;
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      error: null,
+      isLoaded: false,
+      id: props.id,
```
[the remark this answers is not in the document]
Comment (yarosz) [reply client/App.jsx:27]: I think you found it in the index.js file :)
I can't take credit for this, it was @christopher-tucker 's idea :)

File: client/App.jsx [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+      return (
+        <>
+          <GlobalStyle />
+          <h1>Hello from ZuKickstarter!🍕</h1>
+          <h3>This is product name: {data.id}</h3>
+          <Nav />
+          <div>
+            <Story story={data.story}/>
+          </div>
+          <div id="risks">
```
[the remark this answers is not in the document]
Comment (yarosz) [reply client/App.jsx:65]: Yes, I think I did do this in a subsequent commit!

File: client/components/Nav.jsx [hunk 5]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Scrollchor from 'react-scrollchor';
+import { NavButton as Button } from '../StyledDiv';
+
+
+const Nav = (props) => (
+  <ul>
+    <li>
+      <Scrollchor to="#story"><Button>STORY</Button></Scrollchor>
```
[the remark this answers is not in the document]
Comment (yarosz) [reply client/components/Nav.jsx:9]: It's doing a smooth scroll animation, but I really wanted to try and use [ref = React.createRef()](https://reactjs.org/docs/refs-and-the-dom.html) in combination with [.scrollIntoView()](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView#Browser_compatibility) but it looked intimidating so I gave in and used this package. 

Would it work to do the scrolling animation as simply as you described?

[the remark this answers is not in the document]
Comment (quinnfinite) [reply client/components/Nav.jsx:9]: I don't recall it being a slow scroll, but I don't recall off-hand how to alter it.

File: client/components/Story.jsx (hunk 1 above)
Comment (yarosz) [reply client/components/Story.jsx:6]: Now that you point it out, I actually just refactored it to consolidate those lines into one: const Story = ({story}) => {

**Pull request merged**